k8s/pkg/apis/longhorn/v1beta2/instancemanager.go

```diff
@@ -20,6 +20,18 @@ const (
 	InstanceManagerStateUnknown  = InstanceManagerState("unknown")
 )
 
+const (
+	InstanceManagerConditionTypeReady = "Ready"
```

Comment:
Using `Ready` is weird. From the implementation, although the IM is not running, the Ready can be true. It is quite misleading.

Reply:
Renamed to be `Running` as the `longhorn.InstanceManagerStateRunning`